pages/en/billing.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Billing on Subgraph Studio
+---
+
+### Overview
+
+For a quick demo of how billing works on the Subgraph Studio, check out the video below:
+
+<VideoEmbed youtube="UrfIpm-Vlgs" />
+
+Invoices are statements of payment amounts owed by a customer and are typically generated on a weekly basis in the system. You'll be required to pay fees based on the query fees you generate using your API keys. While The Graph protocol operates on Ethereum Mainnet, the billing contract lives on the [Polygon](https://polygon.technology/) network to reduce transaction times and cost. It'll allow you to:
+
+- Add and remove GRT from your wallet
+- Keep track of your balances based on how much GRT you have added to your account, how much you have removed, and your invoices
+- automatically pay invoices based on query fees generated, as long as there is enough billing balance in your account.
+
+There are 4 states your invoice can be in:
+
+1. Pending - the invoice is in the queue to be paid from your billing balance
+2. Paid - your invoice has been successfully paid
+3. Unpaid - there is not enough GRT in your billing balance
+4. Partially Paid - only part of the invoice was paid, since there was not enough billing balance
+
+**See the diagram below for more information:**
+
+![Billing Flow](/img/billing-flow.png)
+
+### Adding GRT to Account
+
+Before making queries on The Graph Network, you will need to add GRT to your account. To add GRT to your account, take the following steps:
+
+1. Purchase GRT and ETH on an exchange of your choice
+2. Send the GRT and ETH to your wallet
+3. Bridge GRT to Polygon using the Subgraph Studio UI
+
+   a) You will receive 0.001 Matic within a few minutes of sending GRT to the Polygon bridge. You can monitor the transaction by inputting your wallet address into the search bar on [Polygonscan](https://polygonscan.com/).
+
+4. Add bridged GRT to the billing contract on Polygon. The billing contract address is: [0x10829DB618E6F520Fa3A01c75bC6dDf8722fA9fE](https://polygonscan.com/address/0x10829DB618E6F520Fa3A01c75bC6dDf8722fA9fE).
+
+   a) In order to complete step #4, you'll need to switch your network in your wallet to Polygon. You can add Polygon's network by connecting your wallet and clicking on "Choose Polygon (Matic) Mainnet" [here](https://chainlist.org/). Once you've added the network, switch it over in your wallet by navigating to the network pill on the top right-hand side corner. In Metamask, the network is called **Polygon Mainnnet**.
```

Comment:
@nenadjaja do we support adding a new network in Metamask directly from the Subgraph Studio?

Reply:
Mistake. For Polygon specifically - if you don't have it in your wallet (MM and others) we'll add it on your behalf when you click on a button in Studio. cc @azf20 @MichaelMacaulay

Reply:
Got it, thank you!